storefront.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  defaultLocale: "en-US",
+  locales: ["fr-fr", "pl-pl"],
+  defaultChannel: {
+    slug: "default-channel",
+    name: "United States Dollar",
+    currencyCode: "USD",
+  },
+  channels: [
```

Comment:
Same goes for channels, there should be a single source for all existing channels. The `defaultChannel` could use a `slug` instead.